### PR TITLE
Added: Contributing.md file.

### PR DESCRIPTION
**Pull request by rajdeepchakraborty-rc**
# Pull Request Format
## Type of PR
- **Add X in the box to specify the improvement type.**
- [ ] Bug fix
- [ ] Feature enhancement
- [X] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to enhance the documentation and accessibility for contributors by providing a separate **Contributing.md** file, which offers a brief walkthrough to guide them in contributing to the project.

## Screenshots / Videos (if applicable)
**Before:**
- There was no dedicated file for contributors. Instead, the documentation for the project and the contributors was mixed within the **README.md** file, which reflected poor documentation practices.
image

**After:**
- This is the look of the newly added **Contributing.md** file.
image
image

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
In addition to creating the **Contributing.md** file, I have also added a redirect link in the **README.md** file(Contributing Section) to guide contributors to the **Contributing.md** after they finish reading the **README.md**.

Thank you for reviewing my pull request!

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (swaraj-das) [inline Contributing.md:1]: code is well.

**Review by swaraj-das — APPROVED** [review]
Merging it

**Comment by rajdeepchakraborty-rc** [comment]
I think you have to assign me to this PR, since the following PR is not updating on the Hacktoberfest website.
image

**Comment by sanjay-kv** [comment]
@swaraj-das 
the labels using is wrong

image


kindly update this.

**Comment by swaraj-das** [comment]
> @swaraj-das the labels using is wrong
> 
> image
> 
> kindly update this.

Yes sir I edited that.

**Pull request merged**